content/nginxaas-azure/changelog/changelog.md

```diff
@@ -144,4 +140,4 @@ See the [Supported Regions]({{< ref "/nginxaas-azure/overview/overview.md#suppor
 
 - {{< icon-feature >}} **In-place SKU Migration from Standard to Standard V2**
 
-   You can now migrate NGINXaaS for Azure from the Standard plan to the Standard V2 plan without redeploying. We recommend upgrading to the Standard V2 plan to access features like F5 WAF for NGINX and more listen ports. The Standard plan will be retired soon. For migration details, see [migrate from standard]({{< ref "/nginxaas-azure/getting-started/migrate-from-standard.md">}}).
+   You can now migrate NGINXaaS for Azure from the Standard plan to the Standard V2 plan without redeploying. We recommend upgrading to the Standard V2 plan to access features like F5 WAF for NGINX and more listen ports. The Standard plan will be retired soon. For migration details, see [migrate from standard]({{< ref "/nginxaas-azure/change-plan/migrate-from-standard.md">}}).
```

Comment:
```suggestion
   You can now migrate NGINXaaS for Azure from the `Standard` plan to the `Standard V2` plan without redeploying. We recommend upgrading to the `Standard V2` plan to access features such as F5 WAF for NGINX and additional listen ports. The `Standard` plan will be retired soon. For migration steps, see [Migrate from Standard]({{< ref "/nginxaas-azure/change-plan/migrate-from-standard.md">}}).
```

I added tick marks around `Standard` and `Standard V2` to match line 131. If the ticks aren't needed, remove in both places.